src/sonic-framework/rebootbackend/rebootbe.h

```diff
@@ -0,0 +1,90 @@
+#pragma once
```

Comment:
Bear in mind `#progma once` has one drawback (aside from being nonstandard), if this file is copied around by the build system to other places, the compiler will think they are different files.

Reply:
I find over 1,000 uses of #pragma in sonic. Thank you!